sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for \
+easy large numeric calculations, but which also can be easily programmed for \
+difficult or long calculations. It can accept a command line \
+argument, in which case it executes that single command and exits. \
+Otherwise, it enters interactive mode. In this mode, it accepts \
+commands one at a time, processes them, and displays the answers. \
+In the simplest case, commands are simply expressions which are evaluated."
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+COPYRIGHT="1999-2014  Landon Curt Noll"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0fcf60917efb10aab145f1180c278a3ab63d789aad6d5e5f066a4458b5f9b692"
+PATCHES="calc$secondaryArchSuffix-$portVersion.patchset"
```

Comment:
Although **`haikuporter -e calc_x86`** generates **`calc_x86-2.12.5.3.patchset`**, you should rename that file to **`calc-2.12.5.3.patchset`**. For your convenience, here's how I would do so:
```
cd /path/to/haikuports/sci-mathematics/calc/patches
git mv calc_x86-2.12.5.3.patchset calc-2.12.5.3.patchset
git commit --amend calc_x86-2.12.5.3.patchset calc-2.12.5.3.patchset
```
Then you should change PATCHES to:
```
PATCHES="calc-$portVersion.patchset"
```
These changes will allow the package to be also built on x86 primary arch and x86_64.